src/rs.cpp

```diff
@@ -3118,8 +3191,8 @@ void rs2_update_firmware_unsigned_cpp( const rs2_device * device,
     // check if the given FW size matches the expected FW size
     if (!val_in_range(image_size, { unsigned_fw_size, unsigned_sr300_size }))
         throw librealsense::invalid_value_exception( rsutils::string::from()
-                                                     << "Unsupported firmware binary image (unsigned) provided - "
-                                                     << image_size << " bytes" );
+            << "Unsupported firmware binary image (unsigned) provided - "
```

Comment:
Please go over the file again and undo places not related to AUS

Reply:
done

Reply:
done